manifests/init.pp

```diff
@@ -82,6 +85,16 @@
     validate_integer($mtu)
   }
 
+  if $omapi_port {
+    validate_integer($omapi_port)
```

Comment:
PR updated to use data types in the class definition for all of the omapi_* parameters.